cdci_data_analysis/flask_app/app.py

```diff
@@ -223,6 +223,16 @@ def remove_nested_keys(D, keys):
     return D
 
 
+def sanitize_dict_before_log(dict_to_sanitize):
+    sensitive_keys = ['token']  # Add any other sensitive keys here
+    sanitized_values = {}
+    for key, value in dict_to_sanitize.items():
+        if key not in sensitive_keys:
+            value = str(value).replace('\n', '').replace('\r', '')
```

Comment:
I would rather replace anything but allowed set of characters.

Reply:
something like using a regex that matches any character that is not a letter, number, or space ?

Reply:
yes